### ISSUE TITLE
Update Photoswipe to latest 5.x version (5.0.6?)

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9454]
# Motivation

We are a major version behind and missing out on some accessibility enhancements. It would be good to get this to the latest update to better align us for the future and for accessibility compliance.

# Additional Info

Previous issue for reference - https://github.com/uiowa/uiowa/issues/4777

Identified as High in initial update audit - https://github.com/uiowa/uiowa/issues/8856

notes from initial audit - Library version bump, open issues (#4777, #6965) and patches

Recent comments from old issue:

https://github.com/uiowa/uiowa/issues/4777#issuecomment-2905197334
>As best as I can tell from looking into this again + re-reading the comments here, it seems that we are in a spot where we can get captions OR grouping to work without custom code. In the current iteration of working on this (https://github.com/uiowa/uiowa/pull/8816) I was able to select the correct field to populate the caption and that just works. However, I did not go through the process outlined in https://github.com/uiowa/uiowa/issues/4777#issuecomment-2140160924 to get the grouping working and I suspect that doing that will cause the captions to fail. Making a note of this so that I can remember when I pick this up again next week. It took me a little while to find the thread above to get the necessary context to recall what had to be done previously.

https://github.com/uiowa/uiowa/issues/4777#issuecomment-3328996774
>Looks like they have provided/merged a caption solution (not token) for this. https://www.drupal.org/project/photoswipe/issues/3453071

**Comment by briand44** [comment]
Grooming conversation from https://github.com/uiowa/uiowa/issues/4777#issuecomment-3666883350

## Grooming
 
**Quadrant:** 

complicated/complex - some unknowns may have been figured out previously but lots to test

 **Discussion:**
 - should we tackle higher value updates (photoswipe?)
 - give us idea of what a high update will take
 - close on getting this updated last time
 - Notes from epic: “library version bump, open issues ([#4777](https://github.com/uiowa/uiowa/issues/4777), [#6965](https://github.com/uiowa/uiowa/issues/6965)) and patches” on why high
 - prior issues with grouping functionality and captions working together
 - they did away with our ability to use the title attribute
   - may have introduced a token as a way of solving for it

**Proposed solution or definition of done:** 
- update photoswipe to latest version
- make sure grouping functionality and captions are able to work together
- make sure we are not introducing regressions
- see if this update resolves issue that Marie is going to comment about

**Comment by e-marie-w** [comment]
Accessibility issue: on pages with image galleries, like https://now.uiowa.edu/news/2025/05/ui-professor-author-melissa-febos-talks-about-books-she-keeps-and-why, when in the gallery modal viewer the images do not have alt text. This is according to the checker from https://www.w3.org/WAI/test-evaluate/easy-checks/image-alt/ and from just inspecting the code. These images do have alt text when using the checker on the regular page, so it's something with the modal specifically.
